doc/pipeline.rst

```diff
@@ -24,10 +24,14 @@ Input file format
 
 The input files for the training and evaluation data should either be in a tabular format with responses as rows and features and score(s) in the columns  in the ``jsonlines`` format with a JSON object per responses on each line. See below for a more detailed description of the ``.jsonlines`` format. 
 
-RSMTool supports input files in ``.csv``, ``.tsv``, ``.sas7bdat``, ``xls``/``.xlsx``, or ``.jsonlines`` format. The format of the file is determined based on the extension. In all cases the :ref:`output files<intermediate_files_rsmtool>` will be saved in ``.csv`` format by default (see :ref:`file format <file_format>` for other intermediate file output options).
+RSMTool supports input files in ``.csv``, ``.tsv``, ``.sas7bdat``, ``.xlsx``, or ``.jsonlines`` format. The format of the file is determined based on the extension. In all cases the :ref:`output files<intermediate_files_rsmtool>` will be saved in ``.csv`` format by default (see :ref:`file format <file_format>` for other intermediate file output options).
 
 For ``Excel`` spreadsheets, all data must be stored in the first sheet. 
 
+.. note::
+
+    RSMTool 8.1 and higher no longer support ``.xls`` files. All Excel files must be in ``.xlsx`` format. 
```

Comment:
Do we want to say why?

Reply:
I thought of it but decided against it: the format is very old by now so I don't think we need to further justify why we no longer support it. We can add something if you think it's important.

Reply:
Fair enough. No ,I think it's fine.